specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -2159,6 +2159,60 @@
           }
         }
       }
+    },
+    "/skus/Spring": {
```

Comment:
>"/skus/Spring" [](start = 4, length = 14)

sorry, I should have been more clear in my original comment.  You only need to include the public facing SKU API in your swagger, not the internal contract with ARM.  See "ARM Sku API For Customer" in the doc linked from the wiki ([https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EUbYj9bCniBNkl0TNibdbRsBb1eSL7dxPHig4yBLmjhE7A?e=KiydcF](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EUbYj9bCniBNkl0TNibdbRsBb1eSL7dxPHig4yBLmjhE7A?e=KiydcF)).  If you search the repos for other implementations of "provider/Microsoft.Whatever/skus" you'll see some examples

Reply:
@pilor, I am a bit confuse here, this api: https://arm-endpoint/subscriptions/{subscription-id}/providers/{resource-provider}/skus?api-version={api-version}&$top={top}$skiptoken={skiptoken}&$filter={filter}  is provided by ARM team. our service team, we provide "/skus/Spring" and "/skus/Spring/apps/deployments" to ARM. We use Swashbuckle  to generate swagger, how can we generate a "provider/Microsoft.Whatever/skus" API in swagger, because we don't have this http route rules.

Reply:
Your swagger defines the public APIs that are exposed underneath your resource provider.  ARM handles the request to {resource-provider}/skus but your clients (SDKs, etc...) should contain the actions that allow the user to retrieve your resource provider's skus.  The internal /skus/Spring, etc... APIs that only ARM calls are not publicly callable by anyone but ARM and should not be in your public swagger.

Reply:
@pilor , thanks for your suggestion. Please review it again.